daml-lf/language/daml-lf.bzl

```diff
@@ -110,15 +112,15 @@ LF_DEV_VERSIONS = [
 ]
 
 # All LF versions
-LF_VERSIONS = [
+LF_VERSIONS = ([
     "1.8",
     "1.11",
     "1.12",
     "1.13",
     "1.14",
     "1.15",
     "2.1",
-] + LF_DEV_VERSIONS
+] if is_intel else ["1.14", "1.15", "2.1"]) + LF_DEV_VERSIONS
```

Comment:
What is the motivation for this change ? 
I would be surprise it does not break anything.

Reply:
If the CI is green, we can still merge and investigate that later.

Reply:
The motivation here is pretty simple: this is required for CI to pass on arm. Note that it does indeed break a lot of things, which are also consequently disabled on arm.

The reason this breaks on arm without this change is because the only way we still support 1.8 in the current build is by relying on `damlc_legacy`, which is a 1.18.0-snapshot release that does not have binaries for arm.

Reply:
I went with incremental trial-and-error to decide what gets disabled on arm - it is possible that there is a better way to do this.

Reply:
make sens.
We probably just have to drop support for old dar. Canton anyway accepts only 1.14 or above. 
I will see that with @moisesackerman-da

Reply:
Thank you!